packages/client/hmi-client/src/components/model/model-parts/tera-model-part-entry.vue

```diff
@@ -30,7 +30,8 @@
 						:options="[
 							{ label: 'Days', value: CalendarDateType.DATE },
 							{ label: 'Months', value: CalendarDateType.MONTH },
-							{ label: 'Years', value: CalendarDateType.YEAR }
+							{ label: 'Years', value: CalendarDateType.YEAR },
+							{ label: 'Weeks', value: CalendarDateType.WEEK }
```

Comment:
Does it matters in the dropdown? Will they be organise properly `Days, Weeks, Months, Years`?

Reply:
I had half a mind of putting the odd one out at the bottom, but sure, I will put the intervals in order